acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeFactory.java

```diff
@@ -139,6 +139,17 @@ public BesuNode createArchiveNodeWithAuthentication(final String name)
             .build());
   }
 
+  public BesuNode createArchiveNodeWithAuthenticationUsingJwtPublicKey(final String name)
```

Comment:
NIT: It doesn't matter what type of node we are creating here. Why don't we call this method `createNodeWithAuthenticationUsingJwtPublicKey` (a bit shorter) :)